### ISSUE TITLE
Multiple major features missing

### ISSUE THREAD

**Issue by Weldoret** [IntelligentSystemGroup-ZZU/PanCAN#1]
There are several major features from the accompanying paper that are missing from this implementation.

Completely missing imports from `__init__.py` at the base of the repo:

- `SlidingWindowAggregator`
- `MultiScaleFusionLayer`
- `RandomWalkAttention` (There's something called `AttentionBasedRandomWalk`, did you mean to import that instead?)

The entire "utils", "datasets", and "scripts" directories are missing, and thus all of the imports from there are errors.

Additionally, we're missing the training code for `tresnet_l` and `cvt_w24`. They are completely unusable despite being valid options for the command line.

There is also no mention in this repository of a RoPE configuration or label grouping procedure, both of which are mentioned in the paper.
